test/unit/controllers/ChoiceControllerSpec.scala

```diff
@@ -89,49 +91,114 @@ class ChoiceControllerSpec extends ControllerLayerSpec with MockCache {
     }
 
     "return 303 (SEE_OTHER)" when {
-      "choice is Arrival" in {
+      "choice is Arrival" should {
         val arrivalForm = JsObject(Map("choice" -> JsString(Arrival.value)))
 
-        val result = controller.submitChoice()(postRequest(arrivalForm))
+        "create cache if empty" in {
+          givenTheCacheIsEmpty()
+          val result = controller.submitChoice()(postRequest(arrivalForm))
 
-        status(result) mustBe SEE_OTHER
-        redirectLocation(result) mustBe Some(routes.ConsignmentReferencesController.displayPage().url)
+          status(result) mustBe SEE_OTHER
+          redirectLocation(result) mustBe Some(routes.ConsignmentReferencesController.displayPage().url)
+          theCacheUpserted.answers mustBe Some(ArrivalAnswers(None, None, None))
```

Comment:
Nit: Aren't `None`s the default values for empty `ArrivalAnswers`?

Reply:
Yes, I made them explicit to reinforce the use of None in each case.